bagitobjecttransfer/recordtransfer/views.py

```diff
@@ -240,6 +245,17 @@ def get_context_data(self, form, **kwargs):
             })
         return context
 
+    def get(self, request, *args, **kwargs):
+        if self.steps.current == 'acceptlegal' and CLAMAV_ENABLED:
+            clamd_socket = clamd.ClamdNetworkSocket(CLAMAV_HOST, CLAMAV_PORT)
+            try:
+                clamd_socket.ping()
+            except clamd.ClamdError as exc:
+                LOGGER.error("Unable to ping ClamAV", exc_info=exc)
+                send_admin_clamav_error(self.request.user, exc)
```

Comment:
In response to your comment on spamming admins with emails - how do you feel about simply removing this feature? I assume we will have a system in our apps when deployed to production that sends an email to _system_ administrators when an ERROR message is found in the logs.

This approach simplifies the app, and handles the case where maybe the site admins are not the actual system admins. In our case, we're both, but that might not be the case for others.

Reply:
We could also add a flag and just display a warning message on the admin pages. I always worry about not getting these notifications fast enough. But I'll remove the emailing for now, we can revisit notifications later.

Reply:
Sure, that sounds good to me